runtime/Js_array.res

```diff
@@ -245,7 +245,6 @@ Js.Array.pop(empty) == None
 ```
 */
 @send
-@return(undefined_to_opt)
```

Comment:
These are fine, as a conversion from undefined to optional seems arbitrary here.